examples/example.py

```diff
@@ -46,14 +54,30 @@ def do_speak(self, arg, opts=None):
             arg = arg.upper()
         repetitions = opts.repeat or 1
         for i in range(min(repetitions, self.maxrepeats)):
-            self.stdout.write(arg)
-            self.stdout.write('\n')
-            # self.stdout.write is better than "print", because Cmd can be
-            # initialized with a non-standard output destination
+            self.poutput(arg)
```

Comment:
Good catch.  I recently converted all of the code within cmd2.py to use .poutput() instead of using stdout.write directly.  I didn't go through all of the examples to make sure they were doing the same however.